### PR TITLE
Move remnants of ocs api requests to v2 endpoint

### PR DESCRIPTION
**Pull request by skjnldsv**
Upcoming `@nextcloud/router` update move the default endpoint to v2, until then, we have to specify it

### CODE REVIEW

File: apps/files/src/components/TransferOwnershipDialogue.vue [hunk 1]
```diff
@@ -172,7 +172,7 @@ export default {
 					},
 				})
 
-				if (response.data.ocs.meta.statuscode !== 100) {
+				if (response.data.ocs.meta.statuscode !== 200) {
```
Comment (nickvergessen) [inline apps/files/src/components/TransferOwnershipDialogue.vue:175]: should not happen, as the axios.get throws on non 20x?

Comment (skjnldsv) [reply apps/files/src/components/TransferOwnershipDialogue.vue:175]: Yeah, I wanted to be safe :shrug:

File: apps/files_sharing/src/components/SharingInput.vue [hunk 2]
```diff
@@ -197,7 +197,7 @@ export default {
 				shareType.push(this.SHARE_TYPES.SHARE_TYPE_EMAIL)
 			}
 
-			const request = await axios.get(generateOcsUrl('apps/files_sharing/api/v1') + 'sharees', {
+			const request = await axios.get(generateOcsUrl('apps/files_sharing/api/v1', 2) + 'sharees', {
```
Comment (nickvergessen) [inline apps/files_sharing/src/components/SharingInput.vue:200]: this now throws but there is no try-catch around it that could show/log a useful error?

File: apps/files/src/components/TransferOwnershipDialogue.vue (hunk 1 above)
Comment (nickvergessen) [reply apps/files/src/components/TransferOwnershipDialogue.vue:175]: right, its okay here as there is a try-catch already

File: apps/files_sharing/src/components/SharingInput.vue [hunk 3]
```diff
@@ -287,14 +287,14 @@ export default {
 		async getRecommendations() {
 			this.loading = true
 
-			const request = await axios.get(generateOcsUrl('apps/files_sharing/api/v1') + 'sharees_recommended', {
+			const request = await axios.get(generateOcsUrl('apps/files_sharing/api/v1', 2) + 'sharees_recommended', {
```
Comment (nickvergessen) [inline apps/files_sharing/src/components/SharingInput.vue:290]: also no try-catch here

File: apps/files/src/components/TransferOwnershipDialogue.vue (hunk 1 above)
Comment (skjnldsv) [reply apps/files/src/components/TransferOwnershipDialogue.vue:175]: Ah, right, that's why there was this! I forgot! v2 properly throws!

**Review by ChristophWurst — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Please resolve conflicts

**Review by MorrisJobke — APPROVED** [review]
Code looks good 👍

**Pull request merged**